cli/internal/run/run.go

```diff
@@ -796,7 +797,7 @@ func (e *execContext) exec(pt *nodes.PackageTask, deps dag.Set) error {
 		return nil
 	}
 	// Cache ---------------------------------------------
-	taskCache := e.runCache.TaskCache(pt, hash)
+	taskCache := e.runCache.TaskCache(pt, hash, e.colorCache)
```

Comment:
Pass the initialized colorCache instead of creating a new one so we get the same colors.

Reply:
If this stays constant throughout the run, maybe the reference should be on `RunCache` rather than `TaskCache`? Or maybe instead of sharing the `colorCache`, we should be sharing the prefix with the `TaskCache`?

Reply:
Good call, I think the `RunCache` makes the most sense here.